bitnami/mariadb-galera/pwd.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mariadb-galera-sec
+type: Opaque
+data:
+  mariadb-root-password: bXlSb290UGFzc3dvcmQK
+  mariadb-password: bXlEYlBhc3N3b3JkCg==
+  mariadb-galera-mariabackup-password: bXlCYWNrdXBQYXNzd29yZAo=
```

Comment:
unintended, right? Please remove it from the PR

Reply:
Yes, this was unintended.